packages/expo-sensors/src/Pedometer.ts

```diff
@@ -28,6 +28,10 @@ export type PedometerUpdateCallback = (result: PedometerResult) => void;
  * provided with a single argument that is [`PedometerResult`](#pedometerresult).
  * @return Returns a [`Subscription`](#subscription) that enables you to call
  * `remove()` when you would like to unsubscribe the listener.
+ *
+ * > Pedometer updates will not be delivered while the app is in the background. As an alternative, on Android, use another solution based on
+ * > [`Health Connect API`](https://developers.google.com/fit) or [`Google Fit API`](https://developer.android.com/health-and-fitness/guides/health-connect).
```

Comment:
@behenate The links are switched up - it should be the other way around. Also, it might be worth not recommending Google Fit API because as of right now, it is not even possible to sign up for the API (it is deprecated)